library/include/rocwmma/internal/blend_impl.hpp

```diff
@@ -279,6 +279,7 @@ namespace rocwmma
             using UnpackWordLo   = PermWord<0u, 2u>;
             using UnpackWordHi   = PermWord<1u, 3u>;
             using UnpackByteLoHi = PermByte<0u, 6u, 1u, 7u>;
+            using UnpackByteHiLo = PermByte<3u, 7u, 3u, 7u>;
```

Comment:
Meaning of this transform is not quite fitting, considering the offsets.
Suggestion: "UnpackByte3BCast = PermByte<3u, 7u, 3u, 7u>"